DeviceManager/DeviceManager/View/DashBoard/DashBoardVC.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  DashBoardVC.swift
+//  DeviceManager
+//
+//  Created by Karthik UK on 06/09/19.
+//  Copyright © 2019 Karthik UK. All rights reserved.
+//
+
+import UIKit
+
+class DashBoardVC: UITabBarController {
```

Comment:
Rename files to appropriate names, a tabbar controller should say its a tabbar in name